tests/common/include/tests/common/tools/ray_scan_utils.hpp

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
Are  the changes in this file relevant with the json geometry?. If not, could you make another PR for this file